backend/routers/roadmap.py

```
@@ -95,7 +100,8 @@ async def generate_roadmap(
     except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)